modules/dr-restoring-cluster-state.adoc

```diff
@@ -290,20 +290,15 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 ----
 +
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
-+
-[NOTE]
-====
-Perform the following step only if you are using `OVNKubernetes` network plugin.
-====
 
-. Delete the node objects that are associated with control plane hosts that are not the recovery control plane host.
+. If you are using the `OVNKubernetes` network plugin, delete the node objects that are associated with control plane hosts that are not the recovery control plane host.
 +
 [source,terminal]
 ----
 $ oc delete node <non-recovery-controlplane-host-1> <non-recovery-controlplane-host-2>
 ----
 
-. Verify that the Cluster Network Operator (CNO) redeploys the OVN-Kubernetes control plane and that it no longer references the non-recovery controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before you proceed with the next step.
+. Verify that the Cluster Network Operator (CNO) redeploys the OVN-Kubernetes control plane and that it no longer references the non-recovery controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before you proceed restarting the Open Virtual Network (OVN) Kubernetes pods on all the hosts in the next step.
```

Comment:
```suggestion
. Verify that the Cluster Network Operator (CNO) redeploys the OVN-Kubernetes control plane and that it no longer references the non-recovery controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before you proceed to restart the Open Virtual Network (OVN) Kubernetes pods on all of the hosts in the next step.
```

Also I'm not familiar with OVN, should "OVN-Kubernetes" in the first sentence be in backticks?

Reply:
Typically it is not, across the repo. However it seems like the spelled out "Open Virtual Network (OVN) Kubernetes" should be in the first sentence if that's the first appearance in this module, versus it being in a subsequent instance.